Makefile

```diff
@@ -16,7 +16,7 @@ clean:
 	rm -f ./build.log
 
 .PHONY: build
-build:
+build: update_submodules
```

Comment:
Minor, but looks like this should be on a new line for consistent formatting.

Reply:
https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html
If it's a prerequisite target then it has to go on same line, won't run otherwise.

Reply:
Ah ok 👍